src/components/clusterDetail/ConsoleModal.css

```diff
@@ -0,0 +1,3 @@
+.console-modal-expandable {
```

Comment:
It should be possible to avoid using custom styles by using `Stack` layout for the expandable items https://www.patternfly.org/v4/layouts/stack#with-gutter